torch/csrc/jit/autodiff.cpp

```diff
@@ -597,17 +596,6 @@ static std::vector<Value*> gradientForNode(Node* node, ArrayRef<Value*> grad_val
       });
       return {backward_value->node()->output(0), nullptr, nullptr, nullptr, nullptr};
 
-    } else if (node->matches("aten::log_softmax(Tensor self, int dim) -> Tensor")) {
```

Comment:
This is already working fine through dispatch and derivatives.yaml. Rather than updating here, simply dropping the override.

Reply:
Nm, this seems to be used in symbolic. diff. Will bring back.